README.md

```diff
@@ -9,10 +9,10 @@
 
 ## What's Golem and yapapi?
 
-**[The Golem Network](https://golem.network)** fosters a global group of creators building ambitious software solutions that will shape the technological landscape of future generations by accessing computing resources across the platform. Golem Network is an accessible, reliable, open access and censorship-resistant protocol, democratizing access to digital resources and connecting users through a flexible, open-source platform.
+**[The Golem Network](https://golem.network)** fosters a global group of creators building ambitious software solutions that will shape the technological landscape of future generations by accessing computing resources across the platform. Golem Network is an accessible, reliable, open and censorship-resistant protocol, democratizing access to digital resources and connecting users in a flexible and open-source manner.
```

Comment:
about the second sentence - what is "open-source manner"? ... 

I'd use the word "environment" here ... so that would be "connecting users in a flexible, open-source environment" .. though, honestly, maybe everything after "digital resources" is unnecessary...

Reply:
> open-source manner

Means that the things you use don't have a proprietary licenses or closed source, we try to open-source everything

Reply:
yeah, I know but to me to "connect users in an open-source manner" sounds wrong...